data/questions_gen_postgres.csv

```diff
@@ -128,10 +128,10 @@ Get the cities in the United States and their population,"SELECT city_name, popu
 What is the total number of restaurants serving each type of food?,"SELECT restaurant.food_type, COUNT(DISTINCT restaurant.id) AS total_number_of_restaurants FROM restaurant GROUP BY restaurant.food_type;",restaurants,group_by,
 What is the total count of restaurants in each city?,"SELECT location.city_name, COUNT(DISTINCT location.restaurant_id) AS total_count FROM LOCATION GROUP BY location.city_name;",restaurants,group_by,
 What is the average rating of restaurants serving each type of food?,"SELECT restaurant.food_type, AVG(restaurant.rating) AS average_rating FROM restaurant GROUP BY restaurant.food_type ORDER BY average_rating DESC NULLS LAST;",restaurants,group_by,
-How many restaurants serve Italian food in each city?,"SELECT location.city_name, COUNT(*) AS number_of_restaurants FROM LOCATION JOIN restaurant ON location.restaurant_id = restaurant.id WHERE restaurant.food_type ILIKE '%Italian%' GROUP BY location.city_name ORDER BY number_of_restaurants DESC NULLS LAST;",restaurants,group_by,
+How many restaurants serve Italian food in each city?,"SELECT restaurant.city_name, COUNT(*) AS number_of_restaurants FROM restaurant WHERE restaurant.food_type ILIKE '%Italian%' GROUP BY restaurant.city_name ORDER BY number_of_restaurants DESC NULLS LAST;",restaurants,group_by,
 How many restaurants are there in each city? Order the results by the number of restaurants in descending order.,"SELECT location.city_name, COUNT(DISTINCT location.restaurant_id) AS number_of_restaurants FROM LOCATION GROUP BY location.city_name ORDER BY number_of_restaurants DESC NULLS LAST;",restaurants,group_by,
-Which street has the most number of restaurants?,SELECT DISTINCT location.street_name FROM LOCATION WHERE street_name = (SELECT street_name FROM LOCATION GROUP BY 1 ORDER BY COUNT(restaurant_id) DESC LIMIT 1);,restaurants,order_by,
-Which restaurants serve Italian cuisine or are located in New York? Order the results by the restaurant name.,SELECT restaurant.name FROM restaurant JOIN LOCATION ON restaurant.id = location.restaurant_id WHERE restaurant.food_type ILIKE '%Italian%' OR location.city_name ILIKE '%New York%' ORDER BY restaurant.name NULLS LAST;,restaurants,order_by,
+Which street has the most number of restaurants?,SELECT street_name FROM location GROUP BY street_name ORDER BY COUNT(restaurant_id) DESC LIMIT 1;,restaurants,order_by,
+Which restaurants serve Italian cuisine or are located in New York? Order the results by the restaurant name.,SELECT name FROM restaurant WHERE food_type ILIKE '%Italian%' OR city_name ILIKE ‘%New York%’ ORDER BY name NULLS LAST;,restaurants,order_by,

```

Comment:
I think you used a different quote character that isn't the usual `'`
```
restaurants=# SELECT name FROM restaurant WHERE food_type ILIKE '%Italian%' OR city_name ILIKE ‘%New York%’ ORDER BY name NULLS LAST;
ERROR:  syntax error at or near "York"
LINE 1: ...d_type ILIKE '%Italian%' OR city_name ILIKE ‘%New York%’ ORD...
```

Reply:
Phew good catch!